### PR TITLE
Update workshop page

### LINKED ISSUE #39: Tidy up Workshop page
I think we can tidy up the workshops page https://bitcoinops.org/workshops/.

Suggestions:
- start with a more general introduction on our workshops, and then have summary details of the SF workshop below
- emphasize that the workshops are Chatham House rules to encourage open and candid conversations, but that general findings may be published and fed back to the open source community (we've already had two requests to info@ asking for recordings/streamings, which is a reasonable request. It'd be helpful to explain our reasoning for not offering those).
- add a call to action.
    - companies to be involved
    - companies to host
    - open source developers to attend (or perhaps speak if we change the format)
    - suggestions for locations/topics

### PR DESCRIPTION
**Pull request by jnewbery**
Adds early announcement for workshop #2 in Paris and tidies up the page.

Closes #39

### CODE REVIEW

File: workshops.md [hunk 1]
```diff
@@ -3,42 +3,56 @@ layout: page
...
+  items are distributed to participants but not beyond. Participants are
+  free to share discussion details internally at their companies and publicly,
+  but should refrain from attributing any particular statement to a given
+  individual (Chatham House Rules).
+
+## Workshop #2 - Paris, Fall 2018
+
+We are in the early stages of planning the second Bitcoin Optech workshop in
+Paris for Fall 2018. Member companies who would like to attend should
+[contact us][optech email]!
```
Comment (jamesob) [inline workshops.md]: Looks like this isn't rendering properly; may need to add the `optech email` footnote def?

**Comment by jamesob** [comment]
Looks :+1: aside from the link issue.

File: workshops.md (hunk 1 above)
Comment (jamesob) [reply workshops.md]: Nevermind! Should've tACK'd.

**Comment by jamesob** [comment]
utACK

File: workshops.md (hunk 1 above)
Comment (harding) [reply workshops.md]: Nah, this is fine (tested preview); that link is defined in references.md.  The problem was the image added in last newsletter doesn't have an alt tag defined, so all the builds based on master are failing.  I'll PR a fix for that.

**Comment by jnewbery** [comment]
Rebased on master to resolve Travis failure.

**Comment by jnewbery** [comment]
ping @moneyball

**Comment by moneyball** [comment]
tACK

**Pull request merged**